packages/react/src/lib/renderComponent.tsx

```diff
@@ -231,6 +231,7 @@ const renderComponent = <P extends {}>(
   const direction = rtl ? 'rtl' : 'ltr'
   const felaParam = {
     theme: { direction },
+    disableAnimations,
```

Comment:
We have check in the plugin for this:
```
if (props && props.disableAnimations && type === 'RULE')
```

Do we really need this one here too?

Reply:
But the `felaParam` here is the `props` in the plugin. That was the cause of the issue.
We originally passed or params to fela but that changed in #1957 and it caused this bug.

Reply:
Got it! Thanks for the clarification